src/main/java/org/apache/maven/plugin/resources/remote/AbstractProcessRemoteResourcesMojo.java

```diff
@@ -350,6 +350,14 @@ public abstract class AbstractProcessRemoteResourcesMojo extends AbstractMojo {
     @Parameter(defaultValue = "${project.build.outputTimestamp}")
     private String outputTimestamp;
 
+    /**
+     * Indicate if project workspace files with teh same name should be used instead of from bundle.
```

Comment:
teh --> the
instead of the ones from the bundle

Reply:
done